Makefile

```diff
@@ -108,8 +108,10 @@ clean: ## clean up binaries
 	@echo "🐳 $@"
 	@rm -f $(BINARIES)
 
-install: $(BINARIES) ## install binaries
-	@echo "🐳 $@"
+install: ## install binaries
+	@ls $(BINARIES) > /dev/null 2>&1 || \
```

Comment:
`bin/$$i`depends on many packages, not only on `cmd/$$i `. So I'm not sure comparing `cmd/$$i -ot bin/$$I` is correct.